openedx/core/djangoapps/course_apps/toggles.py

```diff
@@ -19,9 +19,28 @@
     f'{COURSE_APPS_WAFFLE_NAMESPACE}.proctoring_settings_modal_view', __name__
 )
 
+# .. toggle_name: course_apps.exams_ida_enabled
+# .. toggle_implementation: CourseWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Slowly replace edx-proctoring api calls with calls to edx-exams
```

Comment:
I think this flag is to enable testing of exams-ida use rather than slow replacement, we'll stick this everywhere and turn it on for test courses - @Zacharis278 would know better

Reply:
I think the flag itself simply indicates if the exam IDA exists so I think the descriptions should stop at that. Replacing certain calls is just one way this flag is going to be used.

Reply:
ah sounds good, i was going off of the ticket wording [here](https://2u-internal.atlassian.net/browse/MST-1520). Does "Uses exams IDA to handle API calls" sound like a good description @Zacharis278?